src/test/java/core/basesyntax/strategy/operations/BalanceOperationTest.java

```diff
@@ -0,0 +1,44 @@
+package core.basesyntax.strategy.operations;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.dao.FruitDao;
+import core.basesyntax.dao.FruitDaoImpl;
+import core.basesyntax.db.StorageFruits;
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.FruitTransaction;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+public class BalanceOperationTest {
+    private BalanceOperation balanceOperation;
+    private FruitDao fruitDao;
+
+    @Before
+    public void setUp() {
+        fruitDao = new FruitDaoImpl();
```

Comment:
why do you save this into field
from what i see you just pass it into balance handler